workers/unity/Packages/com.improbable.gdk.tools/.CodeGenerator/GdkCodeGenerator/Templates/UnityComponentDataGenerator.tt

```diff
@@ -377,6 +379,7 @@ var fieldDetails = fieldDetailsList[i]; #>
                 get => <#= componentNamespace #>.ReferenceTypeProviders.UpdatesProvider.Get(handle);
```

Comment:
Does this make the `UpdatesProvider` unused? If so, can it be blocked out as well?

Reply:
Will test 👍